src/pages/Content/utiles/utiles.js

```diff
@@ -1,15 +1,54 @@
-// create new url with isqa=true
+
+//Create new url with isqa=true
 export function createQAUrl(address) {
   const url = new URL(address);
-  url.searchParams.append("isqa","true");
+  url.searchParams.append('isqa', 'true');
   return url.toString();
-  }
+}
 
-// set of the special QA Components 
+//Set of the special QA Components
 export const specialQAComp = new Set();
 
-const specialQACompArray = ['wysiwyg.viewer.components.inputs.RadioButton','core.components.MenuButton', 'wysiwyg.viewer.components.Day', 'wysiwyg.viewer.components.BoxSlideShowSlide', 'wysiwyg.viewer.components.StripContainerSlideShowSlide']
+const specialQACompArray = [
+  'wysiwyg.viewer.components.inputs.RadioButton',
+  'core.components.MenuButton',
+  'wysiwyg.viewer.components.Day',
+  'wysiwyg.viewer.components.BoxSlideShowSlide',
+  'wysiwyg.viewer.components.StripContainerSlideShowSlide',
+];
 
-for(let i=0; i<specialQACompArray.length; i++){
+for (let i = 0; i < specialQACompArray.length; i++) {
   specialQAComp.add(specialQACompArray[i]);
-}
\ No newline at end of file
+}
```

Comment:
// Should be ignored when picking component for JIRA ticket
const componentBlacklist = new Set([
  'wysiwyg.viewer.components.inputs.RadioButton',
  'core.components.MenuButton',
  'wysiwyg.viewer.components.Day',
  'wysiwyg.viewer.components.BoxSlideShowSlide',
  'wysiwyg.viewer.components.StripContainerSlideShowSlide',
]);